engine/src/main/java/org/hibernate/validator/resourceloading/AggregateResourceBundleLocator.java

```diff
@@ -88,12 +89,14 @@ public AggregateResourceBundleLocator(List<String> bundleNames, ResourceBundleLo
 	 * contain all entries from all source bundles. In case a key occurs
 	 * in multiple source bundles, the value will be taken from the
 	 * first bundle containing the key.
-	 * @param localesToInitialize The set of locales to initialize at bootstrap.
+	 * @param preloadResourceBundles if resource bundles should be initialized when initializing the locator
+	 * @param localesToInitialize The set of locales to initialize at bootstrap
```

Comment:
In message interpolators, you pass the "preloadResourceBundles" parameter _after_ the "locales" parameter.
Here, you pass the "preloadResourceBundles" parameter _before_ the "localesToInitialize" parameter.

Not sure it's on purpose?

Reply:
Yes it is.